src/e2e/puppeteer/helpers/index.ts

```diff
@@ -73,7 +75,12 @@ const index = <T extends any[]>(...setupArgs: T) => {
   const index = partialWithRef(pageRef, helpers)
 
   beforeEach(async () => {
-    pageRef.current = await setup(...setupArgs)
+    const regex = /\b(drag|drop)\b/i
+
+    // Check if the test name contains 'drag' and 'drop' to determine if the mouse helper should be installed.
+    // TODO: looking for more reliable solution as this forces the use of drag/drop words in the test name.
```

Comment:
Yeah, I don't think this is so good. We need a more imperative way to set this explicitly on a per-test basis, such as a testFlag.

Reply:
Yeah, setting it in testFlag is a better way I think.

Reply:
Aside from the execution, setting up a testFlag has another issue, because `em.testFlags` can only be accessed inside `page.evaluate`, and `mousePointerHelper` then won't be directly used inside `page.evaluate`, instead we have to serialize the function and `eval` it inside the `page.evaluate`.

Reply:
How about change `mousePointerHelper` to a function that can be called in the test itself. Then you don't need any test flag.

Reply:
Yes, I tried that for some reason it does not work unless its called in `setup` function, maybe it has to do something with the `page` instance. I will try to make it work in the test, as that seems to me the most safe option.

Reply:
The other option is to enable it all the time, but only if that does not affect performance, since it would impact all other tests. 

Otherwise, it would be best to enable it by calling a function in the test itself. It doesn't make sense to do it in the setup function if we are only enabling it on a per-test basis. Once you take it out of the `DOMContentLoaded` handler, I don't see why it wouldn't work outside setup.

Reply:
Yes, I took it out of the `DOMContentLoaded` and used `evaluate`, and it is working now although the pointer appears at a delay, I think its better if we call it in `dragAndDropThought` function since it will be invoked if we are performing drag/drop.